bower.json

```diff
@@ -84,8 +84,11 @@
   },
   "resolutions": {
     "moment": "~2.18.1",
-    "patternfly": "~3.24.0",
-    "jquery": "2.1.4",
-    "font-awesome": "~4.7.0"
+    "patternfly": "~3.25.1",
+    "jquery": "3.2.1",
+    "font-awesome": "~4.7.0",
+    "lodash": "4.x",
```

Comment:
A little worried that we need this resolution. Is one of our dependencies relying on lodash 3?